deepchem/dock/pose_generation.py

```diff
@@ -79,6 +79,166 @@ def generate_poses(self,
     raise NotImplementedError
 
 
+class GninaPoseGenerator(PoseGenerator):
+  """Use GNINA to generate binding poses.
+
+  This class uses GNINA (a deep learning framework for molecular
+  docking) to generate binding poses. It downloads the GNINA
+  executable to DEEPCHEM_DATA_DIR (an environment variable you set)
+  and invokes the executable to perform pose generation.
+
+  GNINA uses pre-trained convolutional neural network (CNN) scoring
+  functions to rank binding poses based on learned representations of
+  3D protein-ligand interactions. It has been shown to outperform
+  AutoDock Vina in virtual screening applications [1]_.
+
+  If you use the GNINA molecular docking engine, please cite the relevant
+  papers: https://github.com/gnina/gnina#citation
+  The primary citation for GNINA is [1]_.
+
+  References
+  ----------
+  .. [1] M Ragoza, J Hochuli, E Idrobo, J Sunseri, DR Koes.
+  "Protein–Ligand Scoring with Convolutional Neural Networks."
+  Journal of chemical information and modeling (2017).
+
+  Notes
```

Comment:
Can you change this to "Note" instead? That makes the readthedocs render a nice note blurb